keyboards/handwired/colorlice/keymaps/default/config.h

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 Marhalloweenvt
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#define RGB_DI_PIN B1
```

Comment:
Personally, I'd prefer that this be defined in the keyboard's config, rather than here. 

That way, it doesn't have to be set per keymap, since I suspect it's non-movable

Reply:
Yeah, if you define this here, the keyboard won't work in configurator correctly, so I encourage you to move this to the top level config.h

Reply:
pretty much all these values should go to the keyboard level config.h otherwise configurator won't work correctly.

Reply:
It is very reasonable. I moved it to keyboard config and remove those defines in keymap config